docs/intro.md

```diff
@@ -5,35 +5,51 @@ description: Intro to Rollkit, a modular framework for rollups.
+
+**I- Scalability, Customizability and Sovereignty
 
-## Why is Rollkit necessary?
+Rollups deployed on a monolithic L1 benefit from directly inheriting security from the base layer. But that comes at the cost of scalability, customizability and sovereignty.
+All rollups sharing the same monolithic L1 are competing with each other over the limited block space, resulting in a lack of scalability
+In addition, developers lack the freedom and ability to customize different parts of the stack like the execution environment, putting a limit and constraint on how they build rollups.
+Furthermore, a rollup community on a monolithic L1 doesn't have the ability to fork their chain when needed.
 
-Today, rollups are built either on top of a monolithic base layer or as independent app-chains. Rollups using a monolithic base layer inherit security but lack scalability due to competition for limited block space. They also lack sovereignty to choose and customize execution environment. App-chains are sovereign but require managing security through a decentralized network of validators and are not fully modular as they handle consensus and settlement.
+**II- Security and Time to Market
 
-Rollups built with Rollkit are secured by the base layer, unlike app-chains which require establishing a decentralized network of validators for security. They are scalable because of the modular base layer, which does not have the issue of competing for limited block space as with a monolithic base layer. Additionally, they are sovereign, providing freedom to choose and configure execution environments.
+App-chains might sound like the perfect solution for the problems listed above. While it’s somehow true, app-chains introduced a new set of tradeoffs.
+Developers building their products as app-chains have to bootstrap an entire validator set and face the overhead of managing a full BFT consensus network, increasing the cost and time of bringing their products to market while introducing security risks. 
+Plus, current app-chains aren’t truly modular since they handle data availability, consensus and settlement. And who doesn’t want to build modular? 
 
-Furthermore, building and deploying rollups requires minimal development effort.
 
-<!-- Drafting: elaborate more on security, scalability, sovereignity, and ease of deloyment factors -->
+## Why Rollkit?
 
-## Use Cases
+By deploying on top of specialized data availability layers like Celestia, Rollkit rollups directly inherit security and scalability from the data availability layer => Scalability and Security ✅
 
+Rollkit is built as a modular framework allowing developers to easily customize and deploy rollups => Customizability and Time to Market ✅
+
+Finally, since Rollkit rollups are deployed on modular blockchains like Celestia that separate execution from data availability and consensus, rollups now have the possibility to be sovereign by deploying as app-chain rollup => Sovereignty ✅
```

Comment:
```suggestion
Finally, since Rollkit rollups are deployed on modular blockchains like Celestia that separate execution from data availability and consensus, rollups now have the possibility to be sovereign by deploying as an app-chain rollup => Sovereignty ✅
```